department-of-veterans-affairs/va.gov-cms
# VACMS-998 Uninstall module redirect_after_login

## Description

See #998. 


## QA steps

1. As anon user try to visit the home page http://pr1709.ci.cms.va.gov/
   - [x] Validate that log in screen appears. 

2. Login as selina.cooper@va.gov 
   - [x] Validate that after login, you are on /user/1343 
   - [x] validate you only see the following sections
![image](https://user-images.githubusercontent.com/5752113/81328872-9ee61280-906b-11ea-9db7-1563ed307eb9.png)
   - [x] Validate that when you try to go to the home page  http://pr1709.ci.cms.va.gov/ you end up redirected to /user/1343

3. Logout by going to http://pr1709.ci.cms.va.gov/user/logout
4. Login as yourself as an admin user.  
   - [x] Validate that you end up on your user page  and you see a complete hierarchy of sections.
   - [x] Validate that when you try to go to the home page  http://pr1709.ci.cms.va.gov/ you end up redirected to your user page.


## Definition of Done
- [x] QA'd
- [ ] Product release notes written
